src/schedule/schedule.c

```diff
@@ -75,3 +76,33 @@ void scheduler_init(int type, const struct scheduler_ops *ops, void *data)
 
 	scheduler_register(sch);
 }
+
+void scheduler_get_task_info(struct scheduler_props *scheduler_props,
+			     uint32_t *data_off_size,
+			     struct list_item *tasks, char *data)
+{
+	/* TODO
+	 * - container_of(tlist, struct task, list)->uid->id could possibly be used as task id,
+	 *   but currently accessing it crashes, so setting the value to 0 for now
+	 */
+	struct task_props *task_props;
+	struct list_item *tlist;
+
+	scheduler_props->core_id = cpu_get_id();
+	scheduler_props->task_count = 0;
+	*data_off_size += sizeof(struct scheduler_props);
+
+	list_for_item(tlist, tasks) {
+		/* Fill SchedulerProps */
+		scheduler_props->task_count++;
+		task_props = (struct task_props *)
+			((uint8_t *)scheduler_props + sizeof(struct scheduler_props));
+
+		/* Fill TaskProps */
+		task_props->task_id = 0;
+		*data_off_size += sizeof(struct task_props);
+
+		/* Left unimplemented */
+		task_props->module_instance_count = 0;
```

Comment:
this is a dummy so far. Not getting any real information

Reply:
Yeah, there's a lot of holes here. The info that reference firmware sends here and info we have here in csof don't seem to be 1 to 1 comparable. Still, telemetry tests use this IPC so I need this one.